packages/trip-form/src/GeneralSettingsPanel/index.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import queryParams from "@opentripplanner/core-utils/lib/query-params";
+import {
+  defaultParams,
+  getQueryParamProperty
+} from "@opentripplanner/core-utils/lib/query";
+import { queryType } from "@opentripplanner/core-utils/lib/types";
+
+import CheckboxSelector from "../CheckboxSelector";
+import DropdownSelector from "../DropdownSelector";
+import commonModes from "../__mocks__/modes"; // FIXME: Replace with ref to configuration.
```

Comment:
I don't think we should be importing from `__mocks__` in the files that make it to the library (i.e., not test or story files). My recommendation would be to make this a constant that is imported from a `consts` file and set this as a default value for a new prop `modes`. I'm not crazy about the `configWrapper` below, especially if all that's needed is the modes field.

Reply:
Oh good catch, I forgot about getting rid of this one.